src/elements/OcBreadcrumb.vue

```diff
@@ -34,6 +58,18 @@ export default {
       default: false,
     },
   },
+  computed: {
+    $_ocBreadcrumb_dropdownItems() {
+      if (this.items.length <= 1 || !this.items) return false
+
+      return [...this.items].reverse().slice(1)
+    },
+    $_ocBreadcrumb_currentFolder() {
+      if (this.items.length === 0 || !this.items) return { text: "Home" }
```

Comment:
untranslated text - either drop it or add a property where we can set this text from outside. THX